ros2interface/ros2interface/verb/list.py

```diff
@@ -46,24 +46,26 @@ class ListVerb(VerbExtension):
     """List all interface types available."""
 
     def add_arguments(self, parser, cli_name):
-        parser.add_argument('-m', '--only-msgs', action='count',
-                            help='Print out only the message types')
+        parser.add_argument(
+            '-m', '--only-msgs', action='count',
```

Comment:
The `count` action doesn't seem to be used for counting the number of times the option is passed. Just removing it will make it a simple boolean flag.

Same below.

Reply:
I used the ```count``` action so that I'd be able to call ```ros2 interface -m/s/a``` without having another argument follow the call

Otherwise usage changes and I get this error:
```usage: ros2 interface list [-h] [-m ONLY_MSGS] [-s ONLY_SRVS] [-a ONLY_ACTIONS] ```
```ros2 interface list: error: argument -s/--only-srvs: expected one argument```

Reply:
You want `action='store_true'` for that.